### DISCUSSION TITLE
Video - Making a 30 yr old SpaceOrb 360 work with retrogaming and modern gaming

### DISCUSSION THREAD

**Discussion by gamer85** [thingotron/orb9k_circuitpython#4] [General]
Waned to share this helpful video by Scott Hanselman, an American Web Developer, teacher and speaker. Currently, he is a Partner Program Manager at Microsoft) (See his amazing blog, https://www.hanselman.com/blog/).  I am reposting this video because of its relevance.

https://twitter.com/shanselman/status/1350653474325753856?lang=e
https://user-images.githubusercontent.com/20948864/172741005-8bac20a4-4856-46d9-8151-60f3c4b0f02e.mp4

**Comment by petersem** [comment]
Nice
